nvflare/app_opt/confidential_computing/az_cvm_authorizer.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import subprocess
+
+import jwt
+
+from nvflare.app_opt.confidential_computing.cc_authorizer import CCAuthorizer
+
+AZ_CVM_NAMESPACE = "x-az-cvm"
+
+
+class AZCVMAuthorizer(CCAuthorizer):
+    def __init__(self, attester_binary="AttestationClient", maa_endpoint="sharedeus2.eus2.attest.azure.net"):
+        self.attester_binary = attester_binary
+        self.maa_endpoint = maa_endpoint
+
+    def generate(self):
+        cmd = ["sudo", self.attester_binary, "-a", f"https://{self.maa_endpoint}/", "-o", "token"]
+        result = subprocess.run(cmd, capture_output=True, check=True)
+        token = result.stdout.decode().strip()
```

Comment:
just a note, in the docs we might need to mention which MAA we have tested

Reply:
This is the default.  If we do not specify `-a`, the attester has a built-in maa endpoint.